apollo-router/src/state_machine.rs

```diff
@@ -101,6 +134,47 @@ where
         mut messages: impl Stream<Item = Event> + Unpin,
     ) -> Result<(), FederatedServerError> {
         tracing::debug!("Starting");
+        // Studio Agent Spaceport listener
+        let (tx, mut rx) = tokio::sync::mpsc::channel::<SpaceportConfig>(1);
+
+        tokio::spawn(async move {
```

Comment:
Is it possible to move this somehow into the lifecycle for the otel stuff?

We will try and pull the OTel configuration out into a Plugin shortly, and it'd be good to have the code contained in that area.

No worries if not, but it'd be worth adding a comment then to say that this logic should be moving soon.

If not

Reply:
Added a comment to indicate that it will move post-tower.
done: 78d2004be2d85923bd934ce134e3272b3d113c38